manifest.json

```diff
@@ -22,7 +22,9 @@
             "matches": [
                 "https://github.com/**",
                 "https://bitbucket.org/**",
-                "https://vibinex.com/**"
+                "https://vibinex.com/**",
+                "https://gitlab.com/**"
```

Comment:
Please update the manifest version as well in this. `0.0.14` if I am not wrong.

Reply:
LOGAF: 10/10

Reply:
Updated